### DISCUSSION TITLE
Add AdditionalProperties: false to all OCF Types that are JSONSchema Objects?

### DISCUSSION THREAD

**Discussion by JSv4** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#185] [Q&A] [answered]
I noticed we took a consistent approach of not including the `additionalProperties`: **false** property on our type schemas. I don't recall this being a distinction we discussed. Given our earlier discussion on this point and the decision to lock the properties on OCF objects, I think we should lock OCF types that are JSONSchema "object" types too? If we don't you can add properties on a type, just checked. 

I also checked the behavior of enums and scalar JSONSchema types, and I don't think it's necessary to add `additionalProperties` to those as they already don't let you "add" additional properties. E.g. you can't add options to the enum, and, if we're defining a regex string field, for example, only strings that match that regex pattern are accepted, regardless of the presence of an `additionalProperties` flag.

**Comment by pjohnmeyer** [comment, marked as answer]
TWG 3 May: Agreed we should add this to object types
